content/courses/advanced-ada/chapters/subprograms.rst

```diff
@@ -716,10 +856,135 @@ makes the overload resolution algorithm reasonably efficient.
 Operator Overloading
 --------------------
 
-.. todo::
+We've seen :ref:`previously <Operators>` that we can define custom operators
+for record types. We can also overload operators of derived types. This allows
+for modifying the behavior of operators for certain types.
 
-    Complete section!
+To overload an operator of a derived type, we simply implement a function for
```

Comment:
This is overriding, rather than overloading. Overloading entails multiple subprograms with the same "name" being visible at the same time in the same scope.

Reply:
You're right, I'm mixing things up here. I've just added a new section on operator overriding and reworked this section.